scss/_patterns_form-tick-elements.scss

```diff
@@ -176,12 +189,30 @@
     display: inline;
   }
 
+  // fake tick hover state
+  .p-checkbox__input:hover + .p-checkbox__label,
+  .p-radio__input:hover + .p-radio__label {
+    @extend %vf-pseudo-tick-box-hover;
```

Comment:
The `__input` elements are hidden, and replaces by fake styled one. So maybe it would be easier to do the hover on whole `p-checkbox` and `p-radio` instead?

Reply:
You're right. We do lose the ability to add a hover style for the checked tickbox that way though